docs/source/user_guide/configuration.md

```diff
@@ -0,0 +1,47 @@
+# Configuration
+
+For a complete list of configuration options, see :link: ./env_vars :link-type: doc
+
+## Backend selection
+
+The torch.compile backend can be configured with the `VLLM_SPYRE_DYNAMO_BACKEND` environment variable.
+
+All models can be tested on CPU by setting this to `eager`.
+To run inference on IBM Spyre Accelerators, the backend should be set as:
+
+| Model type | vLLM backend | `VLLM_SPYRE_DYNAMO_BACKEND` configuration | Notes |
+| --- | --- | --- | --- |
+| Decoder | v0 | sendnn_decoder | V0 support for decoder models is deprecated |
+| Decoder | v1 | sendnn_decoder | |
+| Embedding | v0 | sendnn | |
+| Embedding | v1 | N/A | Embedding models are not yet supported on V1 |
+
+## Batching modes
+
+When running decoder models, vLLM Spyre supports a static batching mode and a continuous batching mode.
+
+### Static batching
+
+With static batching, graphs are pre-compiled for the configured batch shapes and each batch must finish processing before a new batch can be scheduled. This adds extra constraints on the sizes of inputs and outputs for each request, and requests that do not fit the precompiled graphs will be rejected.
+
+Static batching mode is enabled by default, and can be explicitly enabled by setting `VLLM_USE_CB=0`.
+
+The batch shapes are configured with the `VLLM_SPYRE_WARMUP_*` environment variables. For example, to warm up two graph shapes for one single large requests and four smaller requests you could use:
+
+```shell
+export VLLM_SPYRE_WARMUP_BATCH_SIZES=1,4
+export VLLM_SPYRE_WARMUP_PROMPT_LENS=4096,1024
+export VLLM_SPYRE_WARMUP_NEW_TOKENS=1024,256
+```
+
+### Continuous Batching
+
+NB: Continuous batching is not currently supported on IBM Spyre Accelerators. A CPU-only implementation is available by setting `VLLM_SPYRE_DYNAMO_BACKEND=eager`. Continuous batching can be enabled with `VLLM_USE_CB=1`
```

Comment:
Maybe this can be highlighted as a [callout](https://myst-parser.readthedocs.io/en/latest/syntax/admonitions.html#admonition-types) instead of using NB

Reply:
ah nice, yeah that's much better